### PR TITLE
Default layout

### PR DESCRIPTION
**Pull request by Riddy999**
# Type of change

- [x] New feature (non-breaking change which adds functionality)

# Description

Implementation of a default Layout with templates and slots
# Checklist:

- [x] I have commented my code (or ChatGPT did), particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings, neither in my IDE nor in my browser
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -3,4 +3,5 @@
 VITE_SETTINGS_API_URL=http://localhost:3000/api/settings/
 VITE_VIVAWALLET_TRANSACTION_ORDER=http://localhost:3000/api/vivawallet/transaction_order/
 VITE_VIVAWALLET_TRANSACTION_VERIFICATION=http://localhost:3000/api/vivawallet/transaction_verification/
-VITE_STRIPE_PUBLIC_KEY=pk_test_51NTszUJEpEVZWKtc2yNFPJ9DYSmhNnf04vCUmbM3fMS94WK2w1YuhiTcxMIti8p3etufbrsr1oJpG2OUaLUmNUTU00cxAmOXLZ
\ No newline at end of file
+VITE_STRIPE_PUBLIC_KEY=pk_test_51NTszUJEpEVZWKtc2yNFPJ9DYSmhNnf04vCUmbM3fMS94WK2w1YuhiTcxMIti8p3etufbrsr1oJpG2OUaLUmNUTU00cxAmOXLZ
+VITE_AUGUSTIN_COVER=https://augustin.or.at/wp-content/uploads/2023/08/Augustin_580_Cover_web.jpg
```
Comment (lebe1) [inline .env:7]: You added "VITE_AUGUSTIN_COVER" as a new variable but it is not used anywhere...

File: src/views/PaymentConfirmation.vue [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
-    <button class="bg-green-600 rounded-full py-3 px-6 text-white font-bold">Logout</button>
-  </div>
+  <component :is="$route.meta.layout || 'div'">
+      <template #main>
+          <div className="grid grid-rows-5 h-full place-items-center">
+              <div className="text-center font-semibold text-4xl">
+                  Symbol zeigen
+              </div>
+              <div class="row-span-2 text-6xl font-bold">
+                <img class="rounded-full object-cover h-60 w-60" alt="Titelblatt" src="https://augustin.or.at/wp-content/uploads/2023/08/Augustin_580_Cover_web.jpg" />
```
Comment (lebe1) [inline src/views/PaymentConfirmation.vue:9]: Assuming you want to use your new variable VITE_AUGUSTIN_COVER over here?

File: src/views/InformationPrintEpaper.vue [hunk 3]
```diff
@@ -1,28 +1,33 @@
...
+            <div className="h-screen px-10 py-16 grid grid-rows-">
+                <div className="place-items-center">
+                    <img alt="Augustin logo" className="logo mx-auto my-2" src="@/assets/logo.svg" width="270"
+                        height="150" />
+                </div>
+                <div className="flex items-center font-semibold text-4xl place-content-center">
+                    E-Paper
+                </div>
+                <div className="flex text-center w-full place-content-center text-xl">
+                    E-Paper ist einen digitale Zeitung, diese wird dir nach deinem Kauf in unserem online Store angezeigt.
```
Comment (lebe1) [inline src/views/InformationPrintEpaper.vue:17]: E-Paper ist eine Option zur digitalen Zeitung. Diese wird dir nach deinem Kauf in unserem Online-Magazin angezeigt.

**Review by lebe1 — COMMENTED** [review]
Two small issues I found which you can fix within a minute

**Review by nanu-c — APPROVED** [review]

**Pull request merged**